### ISSUE TITLE
Change URL field of self posts

### ISSUE THREAD

**Issue by zambonin** [michael-lazar/rtv#153]
Hello,

Do you think adding `self.subreddit_name` instead of `selfpost` to the `data['url']` field would be a welcome addition so to mirror the information displayed on the original site, or would it be classified as duplicate content, and hence, useless to the layout of the application?

I do believe that this is managed on [this](https://github.com/michael-lazar/rtv/blob/master/rtv/content.py#L136) line, and it wouldn't be too big of a change.

Thanks.

[edit] Other cosmetic features such as `/u/` before the username to match the `/r/` before the subreddit, separate the time and the comment counts with a black dot, bracketed tags etc. also could be changed in a whim.

**Comment by michael-lazar** [comment]
Hi, apologies for the delayed response. 

There are two main goals that keep in mind when making formatting changes:

1. The layout should be compact so it can fit on small terminal screens with minimal text wrapping or ugliness.
2. The sections should be easy to parse visually, especially on black/white displays. 

That being said,

- I agree that ``self.subreddit_name`` is probably better than ``selfpost`` for the reason that you gave. Also IMO ``self.subreddit_name`` looks bit nicer. Either way it's just a placeholder for an empty link. Like you said, this would be easy to change.
- I think the ``/u/`` adds unnecessary clutter.
- I can take or leave the second black dot. I left it out because I felt it was unnecessary, but if you think it adds readability it might be worth adding.

If you have additional ideas and are inclined, go ahead and try them out and report back with before/after screenshots!

**Comment by zambonin** [comment]
Hello, no problem with that.

I [added the `self.subreddit_name`](https://cloud.githubusercontent.com/assets/10156699/10473787/bcd37dd4-7205-11e5-95cb-0f1581f993f5.png) bit and also took the liberty of adding brackets in every flair so it's consistent. If that's not to your liking, it can be removed effortlessly.

I have also added an [exit prompt](https://cloud.githubusercontent.com/assets/10156699/10473786/bcd34b2a-7205-11e5-83fe-c8bcc7d3de14.png). Saves the hassle of logging in again if you hit `q` unintentionally.

I refrained from adding the extra black dot. It would lead to creating a `get_separator` method so [this line](https://github.com/michael-lazar/rtv/blob/master/rtv/subreddit.py#L213) could use the `config.unicode` setting...

I also had another idea that consisted of a timeout when messages like `Welcome, user` or `Logged out` appear on screen, so the user would not have to press ENTER after these events. However, my curses knowledge is limited for now, so I'd just like to leave that registered for a later implementation if possible?

Thanks for your time. If you want to see the code, I have forked the project and commited accordingly. I'll make a pull request if everything's OK.
